docs/CONFIG-VARS.md

```diff
@@ -22,6 +22,7 @@ Supported configuration variables are listed in the table below.  All variables
     - [Cert-manager](#cert-manager)
   - [Postgres](#postgres)
     - [External Postgres](#external-postgres)
+  - [Cloud](#cloud)
```

Comment:
Having a section titled Cloud info and then another section titled Cloud is confusing. I suggest combining the new variables into the existing Cloud info section.

Reply:
combined